.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0
```

Comment:
We shouldn't need this.

Reply:
I added the file to .gitignore

Reply:
Since it's in an older commit you should also `git rm` it.